src/Aquifer.Public.API/Endpoints/Bibles/Alignments/Greek/Endpoint.cs

```diff
@@ -0,0 +1,272 @@
+        ORDER BY
+            bvw.WordIdentifier,
+            gntw.WordIdentifier,
+            gntws.GreekSenseId
+        """;
+
+    private const string GreekWordsQuery = """
+        SELECT
+            gw.Id,
+            gw.[Text] AS Word,
+            gw.GrammarType,
+            gw.UsageCode,
+            gl.[Text] AS Lemma,
+            sn.[Value] AS StrongsNumber
+        FROM GreekWords gw
+            JOIN GreekLemmas gl ON gw.GreekLemmaId = gl.Id
+            JOIN StrongNumbers sn ON gl.StrongNumberId = sn.Id
+        WHERE gw.Id IN @greekWordIds
+        """;
+
+    private const string GreekSensesQuery = """
+        SELECT
+            gs.Id,
+            gs.DefinitionShort AS [Definition],
+            STRING_AGG(gsg.[Text], '||') AS Glosses
+        FROM GreekSenses gs
+            JOIN GreekSenseGlosses gsg ON gsg.GreekSenseId = gs.Id
+        WHERE gs.Id IN @greekSenseIds
+        GROUP BY
+            gs.Id,
+            gs.DefinitionShort
+        """;
+
+    public override void Configure()
+    {
+        Get("/bibles/{BibleId}/alignments/greek");
+        Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.OneHourInSeconds));
+        Description(d => d.ProducesProblemFE());
+        Summary(s =>
+        {
+            s.Summary = "Gets a Bible's Greek alignment information.";
+            s.Description =
+                "For a given Bible and book of the Bible, returns the Bible's text along with associated Greek alignment information and Greek sense data for each verse, similar to a reverse interlinear Bible. Data is returned for all verses within the chapter and verse parameters.";
+        });
+    }
+
+    public override async Task HandleAsync(Request request, CancellationToken ct)
+    {
+        var bookCodeEnum = BibleBookCodeUtilities.IdFromCode(request.BookCode);
+        var lowerBounds = new BibleWordIdentifier(bookCodeEnum, request.StartChapter, request.StartVerse, request.StartWord).WordIdentifier;
+        var upperBounds = BibleWordIdentifier.GetUpperBoundOfWord(bookCodeEnum, request.EndChapter, request.EndVerse, request.EndWord).WordIdentifier;
+
+        var bookData = await dbContext.BibleBooks
+            .Where(bb =>
+                bb.Bible.Enabled &&
+                !bb.Bible.RestrictedLicense &&
+                bb.Bible.GreekAlignment &&
+                bb.Bible.Id == request.BibleId &&
+                bb.Code == request.BookCode.ToUpper())
+            .Select(bb => new
+            {
+                BibleId = bb.Bible.Id,
+                BibleName = bb.Bible.Name,
+                BibleAbbreviation = bb.Bible.Abbreviation,
+                BookCode = bb.Code,
+                BookName = bb.LocalizedName,
+            })
+            .FirstOrDefaultAsync(ct);
+
+        if (bookData == null)
+        {
+            await SendNotFoundAsync(ct);
+            return;
+        }
+
+        var greekAlignmentSource = await dbContext.Database.GetDbConnection()
```

Comment:
This query and the ones below are using Dapper (only the last two needed it but I did it for all of them for consistency).  I propose that instead of using raw SQL queries with EF we use Dapper instead for the scenarios where we want to pass straight SQL.  Dapper's usage is similar to the raw SQL EF method call but it provides much more power when we want to improve performance.